docs/source/introduction.md

```diff
@@ -0,0 +1,114 @@
+# Introduction
+
+```{include} ../../README.md
```

Comment:
Wil je dit per se op deze manier includeren? Als het ook anders kan (bv. hard-coded), dan zou ik er geen list item van maken. Wat mij betreft is de bullet point een beetje willekeurig namelijk.